robottelo/config/facade.py

```diff
@@ -119,6 +123,71 @@ def __repr__(self):
             self.__wrapped__,
         )
 
+    def get_project_root(self):
```

Comment:
This is now defined here and in `robottelo.config.base`. I think you should import the already defined function, and move it if that causes a circular import.

Reply:
Fixed! Using it from base right now.